docs/02-for-app-authors/12-linter.md

```diff
@@ -4,3 +4,54 @@ The Flatpak builder lint tool is a linter for Flatpak manifests. It's used to ch
+| `finish-args-arbitrary-dbus-access`         | Generic D-Bus access is requested.                                           | No[^dbus]    |
+| `finish-args-arbitrary-xdg-cache-access`    | Filesystem permission want `xdg-cache`.                                      | Yes          |
+| `finish-args-arbitrary-xdg-config-access`   | Filesystem permission want `xdg-config`.                                     | Yes          |
+| `finish-args-arbitrary-xdg-data-access`     | Filesystem permission want `xdg-data`.                                       | Yes          |
+| `finish-args-flatpak-spawn-access`          | The package requested access to `org.freedesktop.Flatpak`.                   | Yes          |
+| `finish-args-incorrect-dbus-gvfs`           | D-Bus name `org.gtk.vfs` doesn't exist.                                      | No           |
+| `finish-args-not-defined`                   | No `finish-args`. For the case where this is needed put an empty array.      | No           |
+| `finish-args-unnecessary-appid-own-name`    | D-Bus permission for app id name is granted automatically.                   | No           |
+| `finish-args-unnecessary-xdg-cache-access`  | Filesystem permission want `xdg-cache`.                                      | Yes          |
+| `finish-args-unnecessary-xdg-config-access` | Filesystem permission want `xdg-config`.                                     | Yes          |
+| `finish-args-unnecessary-xdg-data-access`   | Filesystem permission want `xdg-data`.                                       | Yes          |
+| `flathub-json-modified-publish-delay`       | Reduced publishing delay in `flathub.json`.                                  | No[^extra]   |
+| `flathub-json-skip-appstream-check`         | Skipping the appstream check in `flathub.json`                               | Yes          |
+| `module-*-source-git-branch`                | A git source has a branch.                                                   | No           |
+| `module-*-source-git-local-path`            | A git source has a file URL.                                                 | No           |
+| `module-*-source-git-no-commit-or-tag`      | A git source has no commit or tag (The `*` is the module name).              | No           |
+| `module-*-source-git-no-url`                | A git source has no URL specified.                                           | No           |
+| `module-*-source-git-url-not-http`          | A git source URL is not http nor https.                                      | No           |
+| `toplevel-cleanup-debug`                    | `/lib/debug` is in the top level `cleanup` rule.                             | No           |
+| `toplevel-no-command`                       | The `command` property is missing.                                           | No[^baseapp] |
```

Comment:
I notice this only has baseapp here, are there no exceptions for runtimes? KDE and Elementary don't have one.  Would be good to note it here too.